packages/react-liveness/src/components/FaceLivenessDetector/service/utils/__tests__/streamProvider.test.ts

```diff
@@ -119,6 +119,22 @@ describe('LivenessStreamProvider', () => {
         mockVideoEl
       );
     });
+
+    test('with credential provider', () => {
+      const credentialProvider = async () => {
+        return {
+          accessKeyId: 'test',
+          secretAccessKey: 'test',
+        };
+      };
```

Comment:
Can we add explicit types here? Want to see what we would need to do for a custom provider (and not use one from the providers offered by the aws-sdk) like you did here